### PR TITLE
Add more helpful content

### PR DESCRIPTION
**Pull request by emranbm**
I've added more new content on the maturity model page, in comparison to the previously-reviewed google-doc. Furthermore, the home page itself has changed a little bit.  
Please kindly review the new content for the homepage and the maturity model page.

### CODE REVIEW

**Comment by emranbm** [comment]
The failure of the Jekyll build job is gonna be fixed in #46.

File: MATURITY_MODEL.md [hunk 1]
```diff
@@ -1,31 +1,58 @@
...
+Qualitic assumes that the product has a _[Continuous Integration](https://www.atlassian.com/continuous-delivery/continuous-integration)_ pipeline, facilitating its maintenance and development process.  
+It's also assumed that the pipeline has at least five (abstract) steps, as mentioned in the next sections. In each item, there is a field named `Applied Stage` that helps where that item best fits to be cared.
 
-+ Baseline CI Pipeline
-+ Maturity Items
-+ Maturity Levels
+#### 1. Commit Stage
+It's the first time a developer introduces a new change to the source code repository. There can be some automatic checks validating the change.  
+Generally, commit stage ensures that a change is worth reviewing by a human-being; since many trivial issues (like compile error) can be caught by machine, instead of a developer.  
+Although, many non-trivial issues also can be checked automatically (with a more complexity, off course), it's recommended to keep this stage fast enough to deliver early feedback to the developer, more rapidly.
```
Comment (mojyou) [inline MATURITY_MODEL.md:11]: of course?

**Review by mojyou — COMMENTED** [review]
Thanks

File: MATURITY_MODEL.md (hunk 1 above)
Comment (emranbm) [reply MATURITY_MODEL.md]: removed the extra 'f'. Thanks.

**Review by mojyou — APPROVED** [review]
Thanks dude.

File: MATURITY_MODEL.md [hunk 2]
```diff
@@ -1,31 +1,58 @@
-# Qualitic Maturity Model
+# QMM, in details
+Qualitic Maturity Model consists of 5 levels; the first level introduces the initial steps to get in the picture of quality; whereas level 5 indicates the ultimate state of quality.
 
-The Qualitic Maturity model consists of three parts.
+## Baseline CI Pipeline
+Qualitic assumes that the product has a _[Continuous Integration](https://www.atlassian.com/continuous-delivery/continuous-integration)_ pipeline, facilitating its maintenance and development process.  
+It's also assumed that the pipeline has at least five (abstract) steps, as mentioned in the next sections. In each item, there is a field named `Applied Stage` that helps where that item best fits to be cared.
```
Comment (littleali) [inline MATURITY_MODEL.md:6]: to be cared -> to care.

File: MATURITY_MODEL.md [hunk 3]
```diff
@@ -1,31 +1,58 @@
...
 
-+ Baseline CI Pipeline
-+ Maturity Items
-+ Maturity Levels
+#### 1. Commit Stage
+It's the first time a developer introduces a new change to the source code repository. There can be some automatic checks validating the change.  
+Generally, commit stage ensures that a change is worth reviewing by a human-being; since many trivial issues (like compile error) can be caught by machine, instead of a developer.
```
Comment (littleali) [inline MATURITY_MODEL.md:10]: Generally, "the" commit.

caught by "a" machine.

File: MATURITY_MODEL.md [hunk 4]
```diff
@@ -1,31 +1,58 @@
...
+Qualitic assumes that the product has a _[Continuous Integration](https://www.atlassian.com/continuous-delivery/continuous-integration)_ pipeline, facilitating its maintenance and development process.  
+It's also assumed that the pipeline has at least five (abstract) steps, as mentioned in the next sections. In each item, there is a field named `Applied Stage` that helps where that item best fits to be cared.
 
-+ Baseline CI Pipeline
-+ Maturity Items
-+ Maturity Levels
+#### 1. Commit Stage
+It's the first time a developer introduces a new change to the source code repository. There can be some automatic checks validating the change.  
+Generally, commit stage ensures that a change is worth reviewing by a human-being; since many trivial issues (like compile error) can be caught by machine, instead of a developer.  
+Although, many non-trivial issues also can be checked automatically (with a more complexity, of course), it's recommended to keep this stage fast enough to deliver early feedback to the developer, more rapidly.
```
Comment (littleali) [inline MATURITY_MODEL.md:11]: Although(remove comma)

Comment (littleali) [reply MATURITY_MODEL.md:11]: With a more-> remove "a"

File: MATURITY_MODEL.md [hunk 5]
```diff
@@ -1,31 +1,58 @@
...
+> One machine can do the work of fifty ordinary men. No machine can do the work of one extraordinary man.  
+<small>_Elbert Hubbard, American writer and philosopher_</small>
+
+#### 2. Code Review
+As mentioned above, not all issues of a suggested change are detected by the commit stage and hence it needs a manual review - usually by a co-worker.
```
Comment (littleali) [inline MATURITY_MODEL.md:16]: stage and -> stage, and

File: MATURITY_MODEL.md [hunk 6]
```diff
@@ -1,31 +1,58 @@
...
+
+#### 4. Performance Test Stage
+Although most user requirements are of functional type, usually there are also some [non-functional](https://www.guru99.com/non-functional-requirement-type-example.html) ones, even if not declared formally. As an example, a simple phone book application is supposed to save a new phone number in a timely manner, even if it's not specified anywhere. (i.e. no one waits for an hour for a single phone number to be saved!)  
+This stage is the place for running non-functional tests, specifically [performance tests](https://www.guru99.com/performance-testing.html).  
+Performance and Acceptance tests can also run in parallel in a single stage.
+
+#### 5. Staging
+The final stage, before releasing the change into production, is named _Staging_. Here the (recently-changed) product is deployed to a production-like environment (with the exact architecture, infrastructure, etc.). It helps identify even more issues missed in the previous stages. Further more, it's a kind of test for the production deployment procedure itself.
```
Comment (littleali) [inline MATURITY_MODEL.md:27]: Further more -> Furthermore

File: MATURITY_MODEL.md [hunk 7]
```diff
@@ -1,31 +1,58 @@
...
+Managing infrastructure, deployment strategies, update policies, etc. are all items of the Environment category.
 
-+ Acceptance Test Stage
+### Build and Continuous Integration
+This category demonstrates concerns about the build, packaging, compatibility, versioning, and so on for the product.
+Preventing compiler warning, unified code style, and static code analysis are examples of items in this category.
 
-+ Performance Test Stage
+### Delivery
+This category is about how the product is delivered to the end users. Currently, there are four metrics for it that are described below.
```
Comment (littleali) [inline MATURITY_MODEL.md:44]: end users -> end-users

File: README.md [hunk 8]
```diff
@@ -1,14 +1,41 @@
...
+## History
+In early 2019, a group of software engineers at **_[Sahab](https://www.sahab.ir)_** started to move from quality as an abstract concept, to an explicit and pragmatic definition. Through this journey, they tried to define (or collect) various maturity items. Some other companies like _[A'van](https://asta.ir)_ and _[Hamravesh](https://hamravesh.com)_ helped to validate the items, to ensure that they're general (yet useful) enough as software quality items.  
+After several sessions, an initial draft of quality items was created. Then the team tried to prioritize these items in the form of 5 levels of maturity.  
+On 12 November 2019, the first conference of Qualitic was held in Tehran, announcing the maturity model, publicly, for the first time.  
+Since then Sahab has actively used QMM and tried effectively to enrich it - through adding, removing, and editing its items.  
 
-## Contribution
+## When not to use QMM?
+Although it's ideal for a product to be developed at highest level of quality, there's no general advice for all products to progress at a specific level; as there are some other factors that affect what level of quality a product needs.
```
Comment (littleali) [inline README.md:25]: at highest -> at the highest

File: README.md [hunk 9]
```diff
@@ -1,14 +1,41 @@
...
-You can contribute to [Qualitic Community](https://github.com/qualiticcommunity) projects on GitHub.
+A very common misconception is that quality takes time and money. Actually, anything you do in a project needs time and money! But, in the long term, quality pays back more than it costs as the project grows. (That's why short-lived projects should refuse some quality practices.)
+
+## Maintenance and Contribution
+QMM is an [open-source project](https://github.com/qualiticcommunity/qualiticcommunity.github.io) and its maintainers highly try to avoid any exercise of authority, while keeping it usable, maintainable, and practical.  
+So contributions of any kind to the maturity model - through opening issues, pull requests, or contacting us any other way - are appreciated.  
+Feel free to let us know how your experience using this maturity model was, and what can make it better for your team/organization.
+
+## Getting started
+Qualitic Maturity Model is described in details [here](/MATURITY_MODEL.html).
```
Comment (littleali) [inline README.md:41]: described in details -> detail (remove s)

**Review by littleali — CHANGES_REQUESTED** [review]

**Review by littleali — APPROVED** [review]

**Pull request merged**